### ISSUE TITLE
chores

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1648]
- replace install script with box ignore check, make sure to add useChrome cache in there also
- fix luminati links in docs

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.13.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.12.0...3.13.0) (2021-05-08)

- build(deps): bump puppeteer from 9.0.0 to 9.1.1 [`#1643`](https://github.com/open-wa/wa-automate-nodejs/pull/1643)
- build(deps): bump socket.io from 4.0.1 to 4.0.2 [`#1647`](https://github.com/open-wa/wa-automate-nodejs/pull/1647)
- fix(URL) [`#1649`](https://github.com/open-wa/wa-automate-nodejs/pull/1649)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.22.0 to 4.22.1 [`#1639`](https://github.com/open-wa/wa-automate-nodejs/pull/1639)
- build(deps-dev): bump @types/node from 15.0.1 to 15.0.2 [`#1640`](https://github.com/open-wa/wa-automate-nodejs/pull/1640)
- build(deps-dev): bump @typescript-eslint/parser from 4.22.0 to 4.22.1 [`#1638`](https://github.com/open-wa/wa-automate-nodejs/pull/1638)
- Release 3.13.0 [`a1e7af8`](https://github.com/open-wa/wa-automate-nodejs/commit/a1e7af819455a8e9e5e313a0b24e1db677e9be05)
- 📝 chore: migrated to `npx ignore-check` #1648 [`86d5f76`](https://github.com/open-wa/wa-automate-nodejs/commit/86d5f76ed31910f3dcef3c9820118d3c9ad51079)
- 🐛 bug [cli]: remove trailing slash on reverse-proxied popups [`7e4bcee`](https://github.com/open-wa/wa-automate-nodejs/commit/7e4bcee56934eddf53ff028352b963ec8cca3a89)
- 📝 chore: updated luminati links to brightdata #1648 [`aaafaa4`](https://github.com/open-wa/wa-automate-nodejs/commit/aaafaa4f1110b2bedaf3f3998113b782d0cb7e75)
- 🔇  remove 'something went wrong' logs [`dad10d9`](https://github.com/open-wa/wa-automate-nodejs/commit/dad10d9dd0938c5c0b7dea9ab93deab039a5940d)
